README.md

```diff
@@ -112,6 +112,29 @@ A loopback text output can be converted with:
 echo -ne "ari:/EXECSET/n=1234;(//1/CTRL/5(//1/EDD/sw-version))" | ./run.sh cace_ari | ./run.sh refda-stdio -l debug | ./run.sh cace_ari
 ```
 
+## Socket-transport Manager
+
+The manager (and others) are built and installed to the local `testroot` with:
+```
+./build.sh
+./build.sh install
+```
+
+The manager can be run from the local installation using:
+```
+./run.sh gdb --args refdm-socket -l debug -a /tmp/mgr.sock
+```
+
+A manual RPTSET can be sent to the manger via command:
+```
+echo -n 821583f61a2d2639a58382250083012205f6 | xxd -r -p | socat STDIO UNIX-SENDTO:/tmp/mgr.sock,bind=/tmp/agent.sock
```

Comment:
The manager's response to this command is:

```
<ERROR> [msg.c:88:cace_amp_msg_decode] Failed to decode AMP version number, error 6
```

Is that expected?

Reply:
There was a typo at the start of the message content, can you confirm that the fixed version works?

Reply:
This one works much better, thanks @BrianSipos